src/main/java/Main.java

```diff
@@ -59,7 +59,7 @@ public static void main(String[] args) throws OWLOntologyCreationException, OWLO
         for(int i=0; i<1; i++) {
             Model scenarioModel = getModelFromGenerator(factory);
 //
-//            scenarioModel = new ScenarioFactory(scenarioModel);
+//            scenarioModel = new ScenarioFactory(scenarioModel)
```

Comment:
Dobra zmiana, dokładnie taka jaka na komentarzach powinna być xD Jak jest lepszy sposób tworzenia scenarioModel (getModelFromGenerator(factory)) to usuń te komentarze, ogólnie jak pr robimy to usuwajmy komentarze, bo nawet jak mieliśmy tam przydatny kod, to najwidoczniej nie był tak przydatny, żeby go kompilować więc można wyrzucić

Reply:
Tylko ten sposób przez scenario factory umożliwia dodawanie np. pieszych przez Model Builder. Tylko to się nie zapisywało do ontologii :/

Reply:
to klepnij osobną funkcję z tym co masz wykomentowane, potem tylko wywołanie będzie podmieniane, nazwij ją odpowiednio, żeby wiadomo było czym się różnią

Reply:
Ok dodałem